bindings/python/utils/conversions.cpp

```diff
@@ -41,15 +41,16 @@ namespace pinocchio
       static SE3 toSE3fromTupleOrList(const TupleOrList & v)
       {
         //bp::extract<SE3::Scalar> to_double;
-        SE3::Quaternion q(
-            (Scalar)bp::extract<Scalar>(v[6]),
-            (Scalar)bp::extract<Scalar>(v[3]),
-            (Scalar)bp::extract<Scalar>(v[4]),
-            (Scalar)bp::extract<Scalar>(v[5]));
-        SE3::Vector3 t(
-            (Scalar)bp::extract<Scalar>(v[0]),
-            (Scalar)bp::extract<Scalar>(v[1]),
-            (Scalar)bp::extract<Scalar>(v[2]));
+	const Scalar& v0 = bp::extract<Scalar>(v[0]);
```

Comment:
Could you align your code here?

Reply:
This is super weird, I don't see this misalignment on my system. And I've tried with 3 different editors, they all show proper alignement

Reply:
The solution is using the Gepetto linter as Crocoddyl does ;)

Reply:
I think this is just in github's merge comparison window that this misalignment appears, https://github.com/stack-of-tasks/pinocchio/blob/e2e70f6b296107ed3a1c2a1fb04c6d6e28f86717/bindings/python/utils/conversions.cpp shows the proper alignment.

Reply:
It's "tabs vs spaces" - the code is committed with a tab, whereas the surrounding lines are using spaces.

Reply:
Yes, I'm changing my editor environment to prefer spaces :/